pkg/testmachinery/config.go

```diff
@@ -71,6 +71,12 @@ const (
 	// TM_TESTRUN_ID_NAME is the name of the environment variable that holds the current testrun id
 	TM_TESTRUN_ID_NAME = "TM_TESTRUN_ID"
 
+	// TM_GIT_SHA_NAME is the name of the environment variable that holds the current git commit sha of the specified location.
+	TM_GIT_SHA_NAME = "GIT_SHA"
+
+	// TM_GIT_REF_NAME is the name of the environment variable that holds the current git ref of the specified location.
+	TM_GIT_REF_NAME = "GIT_REF"
```

Comment:
Any specific reason, why you didn't prefix the new env vars with `TM_`?

Reply:
Not really. My idea was that I only want to sprefix testmachinery specific env vars with `TM` but git has not much todo with the testmachinery:
But I have no problem about changing it to `TM_GIT_xx`.
@dguendisch wdyt?

Reply:
I think adding `TM_` is more consistent. I always viewed `TM_` vars as the TM's "API", as we define the presence and semantics.
Hence I'd vote for adding it.